pyanaconda/modules/payload/payloads/live_image/live_image.py

```diff
@@ -63,16 +63,15 @@ def __init__(self):
         self._verifyssl = True
         self.verifyssl_changed = Signal()
 
-        self._required_space = 1024 * 1024 * 1024
-        self.required_space_changed = Signal()
-
         self._kernel_version_list = []
         self.kernel_version_list_changed = Signal()
 
         self._image_path = conf.target.system_root + "/disk.img"
 
         self._requests_session = None
 
+        self.set_required_space(1024 * 1024 * 1024)
```

Comment:
Why is this default value necessary? Do you know something about it?

Reply:
I was able to find the source of this value and it was added in commit which added live image support. No explanation why this value.

However, when I'm looking into the code I guess we should keep this. It is required when server won't provide us `content-length` in the header. If we don't know this value we can only give a reasonable guess which is this `1G` value.

Reply:
OK, when I'm looking on the new moved code it won't work as it should. I'll change that.